views/jira-configuration.hbs

```diff
@@ -107,161 +107,163 @@
           </div>
```

Comment:
Are all the changes in this file just format/style change?
Can we exclude those in this PR? So that we can keep the PR absolutely focus on its intention.